src/transform/src/use_indexes.rs

```diff
@@ -117,24 +119,37 @@ impl FilterEqualLiteral {
                     }
                     if !expr_to_equivalent_literal.is_empty() {
                         let key_set = &indexes[id];
-                        let mut equivalences = Vec::new();
-                        // find set of keys of the largest size that is a subset of columns
-                        let best_index = key_set
+                        // find all sets of keys for which every column
+                        // corresponds to an equality to a literal
```

Comment:
nit: i think some punctuation in this comment would make it easier to parse, since there's two distinct sentences here